adoptingerlang.org

```diff
@@ -1974,7 +1976,9 @@ Preparing a release to be deployed to production requires different options than
 
 We have two overridden configuration values in the =prod= profile. =dev_mode= is set to =false= so all content is copied into the release directory, we can't utilize symlinks to the =_build= directory from another machine and a production release should be an immutable snapshot of the project. =include_erts= copies the Erlang runtime and the Erlang/OTP applications depended on by the release into the release directory and configures the boot script to point to this copy of the runtime.
 
-The entries added to the configuration are setting =include_src= to =false= and =debug_info= to =strip=. Running the release in production doesn't require the source code, so we set =include_src= to false in order to drop it from the final release to save on space. Additional space is saved by stripping debug information from the beam files with =debug_info= set to =strip=. Debug information is used by tools like the debugger, =xref= and =cover= but in a release those tools won't be used and, unless explicitly included, won't even be available.
+The entries added to the configuration are setting =include_src= to =false=, =debug_info= to =strip= and =_src= versions of the configuration files. Running the release in production doesn't require the source code, so we set =include_src= to false in order to drop it from the final release to save on space. Additional space is saved by stripping debug information from the beam files with =debug_info= set to =strip=. Debug information is used by tools like the debugger, =xref= and =cover= but in a release those tools won't be used and, unless explicitly included, won't even be available.
```

Comment:
> =debug_info= to =strip=, and =_src=

add an oxford comma so that it's more obvious that _src is not used with debug_info